src/sagemaker/estimator.py

```diff
@@ -2501,6 +2502,8 @@ def _get_train_args(cls, estimator, inputs, experiment_config):
         # which is parsed in execution time
         if estimator.enable_network_isolation():
```

Comment:
can we modify this to 
```
        if estimator.enable_network_isolation() is not None:
            train_args["enable_network_isolation"] = estimator.enable_network_isolation()
```

Reply:
Made the change. I had to default it to `False` first, since the problem presents when this keyword argument is not present in train_args. It always has to be in there. Let me know thoughts on that approach.